src/components/ha-push-notifications-toggle.js

```diff
@@ -9,6 +9,7 @@ import "./ha-switch";
 export const pushSupported =
   "serviceWorker" in navigator &&
   "PushManager" in window &&
+  !("externalApp" in window) &&
```

Comment:
This only works for Android. It would be better if we make a data/external.ts file and export `isExternal` as currently defined in `entrypoints/core.ts`

Reply:
I can do that.

Reply:
Also, this doesn't hide it. This just disables it. The whole row is still visible and this change is made in the wrong file.